data/helpers.d/package

```diff
@@ -1,10 +1,34 @@
+# Check if apt is free to use, or wait, until timeout.
+#
+# [internal]
+#
+# usage: ynh_is_dpkg_free
+ynh_is_dpkg_free() {
+    local try
+    # With seq 1 17, timeout will be almost 30 minutes
+    for try in `seq 1 17`
+    do
+        # Check if /var/lib/dpkg/lock is used by another process
+        if sudo lsof /var/lib/dpkg/lock > /dev/null
```

Comment:
Does we really need `sudo` as normally all helper should be called as root ?

Reply:
lsof won't work if it's not root, I added it for my own tests and I forgot to remove it...
Still, It's maybe not useless if an user want to use it on his own.
The problem is, without root, lsof will just don't see any process using the lock. So I'd rather prefer to keep it, even if it's going to be useless in most cases.

Reply:
Note lsof command is here because mariadb is installed. Might we add this requirement in our packages in case of mariadb will be remove in future ?

Reply:
I have this package on my own debian (buster though), but I guess it could be better to have it as a dependency.